.github/workflows/ruff.yml

```diff
@@ -10,4 +10,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: chartboost/ruff-action@v1
\ No newline at end of file
+      - uses: chartboost/ruff-action@v1
+        continue-on-error: true # https://github.com/Draegerwerk/sdc11073/issues/156
```

Comment:
remove this, as it was already done in https://github.com/Draegerwerk/sdc11073/pull/216 when this PR was opened

Reply:
done